js/views/modals/Settings/Page.js

```diff
@@ -30,9 +30,41 @@ export default class extends baseVw {
       'click .js-avatarRight': 'avatarRightClick',
       'click .js-headerLeft': 'headerLeftClick',
       'click .js-headerRight': 'headerRightClick',
+      'change .js-primaryColorPicker': 'handleColorChosen',
+      'change .js-secondaryColorPicker': 'handleColorChosen',
+      'change .js-textColorPicker': 'handleColorChosen',
+      'change .js-highlightColorPicker': 'handleColorChosen',
+      'change .js-highlightTextColorPicker': 'handleColorChosen',
+      'change .js-primaryColorCode': 'handleColorCodeEntered',
+      'change .js-secondaryColorCode': 'handleColorCodeEntered',
+      'change .js-textColorCode': 'handleColorCodeEntered',
+      'change .js-highlightColorCode': 'handleColorCodeEntered',
+      'change .js-textHightlightColorCode': 'handleColorCodeEntered',
     };
   }
 
+  /** Handles when a hex color code is entered by updating color picker. */
+  handleColorCodeEntered(event) {
+    const colorPickerId = $(event.target).data('color-picker-id');
+    const $colorPicker = $(`${colorPickerId}`);
```

Comment:
Optimization has been added - check it out. I think something like what I added would be useful in the base view; what do you think? Naming consistency was updated to match the rest of the view FTMP - good point. The '#' has been added to the ids in the data fields and also the optimization removes this goofyness. I ended up with this post-lint when I switched over to templatized strings.